server/routes/api/chart.py

```diff
@@ -262,7 +262,7 @@ def get_date_range(dates):
     sorted_dates_list = sorted(list(dates))
     date_range = sorted_dates_list[0]
     if len(sorted_dates_list) > 1:
-        date_range = f'{sorted_dates_list[0]} - {sorted_dates_list[-1]}'
+        date_range = f'{sorted_dates_list[0]} to {sorted_dates_list[-1]}'
```

Comment:
is this change strictly necessary? this would require translations for place pages.

how about the [endash](https://www.thepunctuationguide.com/en-dash.html) instead?

Reply:
This isn't necessary but it just might look a little confusing if there is a date range like "2020-03-2020-04". However, there usually aren't date ranges on the place pages so I think I'll just use the endash instead.